pom.xml

```diff
@@ -200,6 +230,22 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>1.4.0</version>
+				<executions>
+					<execution>
+						<phase>integration-test</phase>
+						<goals>
+							<goal>exec</goal>
+						</goals>
+						<configuration>
+							<executable>${basedir}/tacoco-diagnoses</executable>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
I want the integration phase to be TravisCI-first. So i am comfortable with this lopsided approach for now -- unless you think that running `mvn integration-test` is key? Was it part of `mvn install` or `mvn publish`?

Reply:
Github now hosts Maven packages. And it enables you to consume Maven packages much like Maven Central. If anything, i am guessing we can setup projects4testing to be a non-Java (or vanila) Maven package that does not compile, or run or anything... and simply setup us such a package as a `integration` phase dependency for Tacoco.

Reply:
If you run mvn install locally it will also run the integration-test phase, so locally will fail if you didn't clone the projects4testing yet, so how do we want to tackle that?

Reply:
see my other comment. Passing simple -DskipXYZ flags to mvn install does the trick.